apps/librelingo_yaml_loader/tests/test_yaml_loader.py

```diff
@@ -176,20 +176,99 @@ def test_returns_correct_special_characters(self):
             self.fake_values["second_special_character"],
         ]
 
-    def test_returns_correct_settings_audio_files_enabled(self):
-        assert self.result.settings.audio_files_enabled == True
+    def test_returns_correct_settings_audio_settings_enabled(self):
+        assert self.result.settings.audio_settings.enabled == True
 
-    def test_returns_correct_settings_audio_files_disabled(self):
+    def test_returns_correct_settings_audio_settings_tts(self):
+        tts_settings = self.result.settings.audio_settings.text_to_speech_settings
+        assert len(tts_settings) == 1
+        assert tts_settings[0].provider == "Polly"
+        assert tts_settings[0].voice == "Lupe"
+        assert tts_settings[0].engine == "standard"
+
+    def test_returns_correct_settings_audio_disabled(self):
+        new_settings = """
+    Settings:
+        Audio:
+            Enabled: False
+        """
+
+        # Append settings to the file
```

Comment:
this part seems to show up multiple times, maybe it could be extended like this?

```python
append_settings_to_file("""
    Settings:
        Audio:
            Enabled: False
        """)

```